keyboards/planck/keymaps/numist/keymap.c

```diff
@@ -0,0 +1,234 @@
+    XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_1   , KC_2   , KC_3   , XXXXXXX, XXXXXXX,
+    _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_0   , KC_DOT , XXXXXXX, XXXXXXX, XXXXXXX
+),
+
+[_RAISE] = LAYOUT_planck_grid(
+    KC_GRV , KC_1   , KC_2   , KC_3   , KC_4   , KC_5   ,   KC_6 , KC_7   , KC_8   , KC_9   , KC_0   , KC_DEL ,
+    KC_DEL , KC_F1  , KC_F2  , KC_F3  , KC_F4  , KC_F5  ,  KC_F6 , KC_MINS, KC_EQL , KC_LBRC, KC_RBRC, KC_BSLS,
+    _______, KC_F7  , KC_F8  , KC_F9  , KC_F10 , KC_F11 , KC_F12 , KC_HASH, KC_BSLS, KC_PGUP, KC_PGDN, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, XXXXXXX
+),
+
+// Lisa Simpson says: "_LOWER is just S(_RAISE)!"
+[_LOWER] = LAYOUT_planck_grid(
+    KC_TILD, KC_EXLM, KC_AT  , KC_HASH, KC_DLR , KC_PERC, KC_CIRC, KC_AMPR, KC_ASTR, KC_LPRN, KC_RPRN, KC_DEL ,
+    KC_DEL , KC_F1  , KC_F2  , KC_F3  , KC_F4  , KC_F5  , KC_F6  , KC_UNDS, KC_PLUS, KC_LCBR, KC_RCBR, KC_PIPE,
+    _______, KC_F7  , KC_F8  , KC_F9  , KC_F10 , KC_F11 , KC_F12 , KC_TILD, KC_PIPE, KC_HOME, KC_END,  _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, XXXXXXX
+),
+
+};
+
+#ifdef AUDIO_ENABLE
+float plover_song[][2]    = SONG(PLOVER_SOUND);
+float plover_gb_song[][2] = SONG(PLOVER_GOODBYE_SOUND);
+#endif
+
+uint32_t layer_state_set_user(uint32_t state) { return update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST); }
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    if (!process_record_dynamic_macro(keycode, record)) {
+        return false;
+    }
+
+    bool use_cmd = true;
+    if (keymap_config.swap_lalt_lgui == 1) {
+        use_cmd = false;
+    }
+
+    switch (keycode) {
+        case QWERTY:
+            if (record->event.pressed) {
+                set_single_persistent_default_layer(_QWERTY);
+            }
+            return false;
+        case COLEMAK:
+            if (record->event.pressed) {
+                set_single_persistent_default_layer(_COLEMAK);
+            }
+            return false;
+        case PLOVER:
+            if (record->event.pressed) {
+#ifdef AUDIO_ENABLE
+                stop_all_notes();
+                PLAY_SONG(plover_song);
+#endif
+                layer_off(_RAISE);
+                layer_off(_LOWER);
+                layer_off(_ADJUST);
+                layer_on(_PLOVER);
+                keymap_config.raw  = eeconfig_read_keymap();
+                keymap_config.nkro = 1;
+                eeconfig_update_keymap(keymap_config.raw);
+            }
+            return false;
+        case EXT_PLV:
+            if (record->event.pressed) {
+#ifdef AUDIO_ENABLE
+                PLAY_SONG(plover_gb_song);
+#endif
+                layer_off(_PLOVER);
+            }
+            return false;
+        case M_UNDO:
```

Comment:
As above, you don't actually need this. 

You can replace the `AG_NORM` and `AG_SWAP` with `CG_NORM` and `CG_SWAP` respectively, and will get the behavior you want.
https://docs.qmk.fm/#/feature_bootmagic?id=keycodes